Squablr/AppDelegate.m

```diff
@@ -28,36 +25,52 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
     [Parse initializeWithConfiguration:config];
     
+    // Google sign in to restore user signed in state
+    [GIDSignIn.sharedInstance restorePreviousSignInWithCallback:^(GIDGoogleUser * _Nullable user,
+                                                                    NSError * _Nullable error) {
+        if (error) {
```

Comment:
What is this code doing? Seems like throwaway / commented out code again

Reply:
Yeah, the google docs said to include it in the appDelegate file and I'm not too confident how useful having it is since the app has user persistence without it...